### PR TITLE
Run tests using tox if config exists, forward results to sonar

### PR DESCRIPTION
**Pull request by diasdm**
Run tests using tox if config exists, forward coverage to SonarCloud

Tests:
* without tox.ini - https://github.com/MOV-AI/python-delete-me-david/actions/runs/3082597497/jobs/4983692039
* with tox.ini - https://github.com/MOV-AI/python-delete-me-david/actions/runs/3083340740/jobs/4984153082

### CODE REVIEW

File: .github/workflows/py-workflow.yml [hunk 1]
```diff
@@ -159,16 +169,14 @@ jobs:
           -Dsonar.scm.provider=git
           -Dsonar.qualitygate.wait=true
           -Dsonar.qualitygate.timeout=300
+          -Dsonar.python.coverage.reportPaths=coverage.xml
```
Comment (AlexFernandes-MOVAI) [inline .github/workflows/py-workflow.yml:197]: what happens if coverage.xml is not present ?

Comment (diasdm) [reply .github/workflows/py-workflow.yml:197]: SonarCloud shows a warning but I saw no effects on the execution, see [here](https://github.com/MOV-AI/python-delete-me-david/actions/runs/3082597497/jobs/4983692039#step:10:1).
`2022-09-19T14:19:50.7732182Z 14:19:50.772 WARN: No report was found for sonar.python.coverage.reportPaths using pattern coverage.xml`

Comment (AlexFernandes-MOVAI) [reply .github/workflows/py-workflow.yml:197]: ok, just remove verbose option on sonar args

**Review by AlexFernandes-MOVAI — APPROVED** [review]

File: .github/workflows/py-workflow.yml (hunk 1 above)
Comment (diasdm) [reply .github/workflows/py-workflow.yml:197]: Done

**Review by duartecoelhomovai — APPROVED** [review]

**Pull request merged**